components/ProjectPreview/ProjectPreview.js

```diff
@@ -0,0 +1,49 @@
+import Image from "next/image";
+import Link from "next/link";
+import styled from "styled-components";
+
+const ProjectContainer = styled.div`
+  background-color: #faf8f7;
+  margin-left: 20px;
+  padding: 0 10px 10px 10px;
+  margin-bottom: 20px;
+  border-radius: 5%;
+`;
+
+const StyledImage = styled(Image)`
+  height: 100%;
+  width: 100%;
+  border-radius: 5%;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: black;
+`;
+
+export default function ProjectPreview({
+  category,
+  title,
+  imageSource,
+  organizer,
+  width,
+  height,
+  shortDescription,
+  slug,
+}) {
+  return (
+    <StyledLink href={`/projects/${slug}`}>
+      <ProjectContainer>
+        <p>{category}</p>
+        <h2>{title}</h2>
+        <StyledImage
+          src={`/assets/images/${imageSource}`}
+          alt={`Photo ${title} by ${organizer}`}
+          width={width}
```

Comment:
I would recommend to pass a number as value to your width and height props:
```suggestion
          width={200}
```
What value you give is not so important, because it will be overwritten be your style you define in your styled component above.